.github/workflows/mainPipeline.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 3.1.101
+          dotnet-version: '3.1.x'
```

Comment:
I'm not sure I understand; based on the PR title, is your intent for this to be a 5.x version?

Reply:
Was just trying something